fv3core/stencils/basic_operations.py

```diff
@@ -20,18 +21,24 @@ def copy_stencil(q_in: sd, q_out: sd):
         q_out = q_in
 
 
-def copy(q_in, origin=(0, 0, 0), domain=None):
+def copy(q_in, origin=None, domain=None):
```

Comment:
`origin=(0, 0, 0)` is actually safe (since tuples are immutable) and clearer, so long as we're restricting q_in to be a 3-dimensional field. Domain on the other hand has to default to None since its default value is dynamic.